xla/service/gpu/runtime/convolution_thunk.cc

```diff
@@ -22,7 +22,9 @@ limitations under the License.
 #include "absl/status/status.h"
 #include "xla/service/buffer_assignment.h"
 #include "xla/service/gpu/gpu_conv_runner.h"
+#include "xla/service/gpu/stream_executor_util.h"
```

Comment:
This requires a corresponding BUILD file dependency that has this header in its hdrs section.

Reply:
Sorry for the delay, I have fixed it